models/DataObject/Classificationstore/Dao.php

```diff
@@ -64,7 +64,11 @@ public function save()
         $dataTable = $this->getDataTableName();
         $fieldname = $this->model->getFieldname();
 
-        $this->db->delete($dataTable, ['o_id' => $objectId, 'fieldname' => $fieldname]);
+        $dataExists = $this->db->fetchOne("SELECT count(*) c FROM `".$dataTable."` WHERE
+         `o_id` = '".$objectId."' AND `fieldname` = '".$fieldname."' ");
```

Comment:
```suggestion
        $dataExists = $this->db->fetchOne("SELECT o_id FROM `".$dataTable."` WHERE
         `o_id` = '".$objectId."' AND `fieldname` = '".$fieldname."' LIMIT 1");
```
Wouldn't it be faster to just fetch one column with a `LIMIT`?

Reply:
Updated the PR with the suggestion